DISCLAIMER

```diff
@@ -0,0 +1,26 @@
+Apache DataSketches (incubating) is an effort undergoing incubation 
```

Comment:
The name of this file should be DISCLAIMER-WIP. (Ref: http://incubator.apache.org/policy/incubation.html#Releases).

Reply:
renamed